### ISSUE TITLE
Crashes on Android 11

### ISSUE THREAD

**Issue by Lev-VM** [kaii-lb/LavenderPhotos#15]
Experiencing instant crash on OxygenOS 11 (OnePlus 8 Pro), using latest build 0.8.6 beta. Tried to allow access to all files before launching the app - didn't help. Can this be fixed?
![Screenshot_20241227-042200](https://github.com/user-attachments/assets/910bff70-bd2d-4970-97cd-4dcb0816b24a)

**Comment by kaii-lb** [comment]
yeah i know about the issue, [try this build](https://drive.google.com/file/d/1btwf1yKuIFIrxEAcyC0JKmPeOh_YbTx3/view?usp=sharing) and see if it works, if it doesn't launch there should be a log in "Internal Storage/LavenderPhotos"

**Comment by Lev-VM** [comment]
> yeah i know about the issue, [try this build](https://drive.google.com/file/d/13AwhWxtjt4to5TIRnC1p_Faiyw0dN8_1/view?usp=drivesdk) and see if it works, if it doesn't launch there should be a log in "Internal Storage/LavenderPhotos"

Unfortunately this build crashes as well. Also, it doesn't creata a "LavenderPhotos" folder in the root directory and doesn't write the log either even if I create that folder myself prior to launching the app.

**Comment by kaii-lb** [comment]
oh. i cant launch the logcat any earlier than i already am, so can you manually pull logs using `adb logcat --pid=$(adb shell pidof com.kaii.photos)` ?

**Comment by kaii-lb** [comment]
@Lev-VM took way too long but fixed it finally! check latest release

**Comment by Lev-VM** [comment]
Yes, it works now. Thanks!

**Comment by kaii-lb** [comment]
glad! closing as completed
